operator/src/main/java/org/bf2/operator/managers/SecuritySecretManager.java

```diff
@@ -288,15 +289,14 @@ private Secret ssoClientSecretFrom(ManagedKafka managedKafka, Secret current) {
     }
 
     private Secret ssoTlsSecretFrom(ManagedKafka managedKafka, Secret current) {
-        if (!isKafkaAuthenticationEnabled(managedKafka)) {
```

Comment:
Is the isKafkaAuthenticationEnabled check not valid in general?

Reply:
It's valid in other contexts (e.g. building the Kafka listeners), but here it's not enough. Not sure if that's what you're asking.